med-web-app-front/src/components/attachments/upload-attachments.component.js

```diff
@@ -102,16 +144,80 @@ class UploadAttachmentsComponent extends Component {
             selectedFiles: undefined,
             progressInfos: [],
             message: [],
-
             fileInfos: [],
+            dragEnter: false,
         };
     }
 
-    selectFiles(event) {
+    deleteElement(index) {
+        let files = [...this.state.selectedFiles];
+        files.splice(index, 1);
+        if(files.length === 0) files = undefined
+        this.setState({
+            selectedFiles: files,
+        });
+    }
+
+    selectFiles() {
+        this.fileInput.current.click();
+    }
+
+    handleFiles(e) {
+        let selectedFiles = []
+        let files = [...e.target.files]
+        let errorMessage = []
+        if(this.state.selectedFiles !== undefined) selectedFiles = [...this.state.selectedFiles]
+        files.map(file => {
+            selectedFiles.push(file)
+        })
+        if(selectedFiles.length === 0) {
+            selectedFiles = undefined;
+            errorMessage = ["Произошла ошибка попробуйте выбрать файлы снова"]
```

Comment:
```suggestion
            errorMessage = ["Произошла ошибка, попробуйте выбрать файлы снова"]
```
Вообще, если строковые литералы фигачатся прямо в код, предвижу проблемы с локализацией. В идеале сразу бы выбрать библиотеку локализации и использовать её везде, но можно и пока так, раз во всём проекте ничего такого не делалось.